src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="/src/images/imac.jpeg"
+        alt="desktop"
+      />
+      <h4 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h4>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card-review">
+        <div class="card-review__stars">
+          <div class="card-review__star"></div>
+          <div class="card-review__star"></div>
+          <div class="card-review__star"></div>
+          <div class="card-review__star"></div>
+          <div class="card-review__star"></div>
+        </div>
```

Comment:
You need to use this block from the previous task and add the modifier class
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```